vendor_skins/Evercast/libobs/CMakeLists.txt

```diff
@@ -0,0 +1,504 @@
+project(libobs)
+
+find_package(Threads REQUIRED)
+
+find_package(FFmpeg REQUIRED
+	COMPONENTS avformat avutil swscale swresample
+	OPTIONAL_COMPONENTS avcodec)
+include_directories(${FFMPEG_INCLUDE_DIRS})
+
+if (NOT "${FFMPEG_AVCODEC_LIBRARIES}" STREQUAL "")
+	list(REMOVE_ITEM FFMPEG_LIBRARIES ${FFMPEG_AVCODEC_LIBRARIES})
+endif()
+
+if(UNIX)
+	if (NOT APPLE)
+		find_package(X11_XCB REQUIRED)
+		find_package(XCB OPTIONAL_COMPONENTS XINPUT)
+		if (XCB_XINPUT_FOUND)
+			set(USE_XINPUT "1")
+		else()
+			set(USE_XINPUT "0")
+		endif()
+		find_package(PulseAudio)
+		if (NOT "${PULSEAUDIO_LIBRARY}" STREQUAL "")
+			message(STATUS "Found PulseAudio - Audio Monitor enabled")
+			set(HAVE_PULSEAUDIO "1")
+		else()
+			set(HAVE_PULSEAUDIO "0")
+		endif()
+	else()
+		set(HAVE_PULSEAUDIO "0")
+		set(USE_XINPUT "0")
+	endif()
+	find_package(DBus QUIET)
+else()
+	set(HAVE_DBUS "0")
+	set(HAVE_PULSEAUDIO "0")
+	set(USE_XINPUT "0")
+endif()
+
+find_package(ImageMagick QUIET COMPONENTS MagickCore)
```

Comment:
Oh that's fascinatig.  I remember imagemagik from its days of being a PHP vulnerability.  Luckily we don't use it to process random file uploads.